boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/util/SSLUtils.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.boot.microservices.util;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.security.KeyStore;
+
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+
+import com.oneandone.compositejks.CompositeX509KeyManager;
+import com.oneandone.compositejks.CompositeX509TrustManager;
+import com.oneandone.compositejks.KeyStoreLoader;
+import com.oneandone.compositejks.SslContextUtils;
+
+
+/**
+ * Utility methods for SSL context.
+ */
+public final class SSLUtils {
+
+    private static final String X509_ALGORITHM = "SunX509";
```

Comment:
Could you please put some comment, What is and what for we use SunX509 algorithm. Thanks

Reply:
Also, did you consider moving this to IAMConfiguration ?

Reply:
SSL params are moved to '$PA_Home/config/iam/application-proactive.properties'. Comments are added to explain these SSL params.